source/common/usm_pool_config.hpp

```diff
@@ -0,0 +1,80 @@
+//===--- usm_pool_config.hpp -configuration for USM memory pool-------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef USM_POOL_CONFIG
+#define USM_POOL_CONFIG
+
+#include "disjoint_pool.hpp"
+#include "ur_api.h"
```

Comment:
When I tried to include the `usm_pool_config.hpp` file in the test it failed because It could not find the `ur_api.h` so I think this include needs a path perhaps?

Reply:
I forgot to add usm_pool_config to 'common' lib. It should work now in tests, as they already link with common.